src/features/profile/HouseKeepingCard.js

```diff
@@ -57,7 +57,8 @@ const HouseKeepingCard = () => {
 //         )
 //       }
 //     } else {
-// //      const { error } = await dispatch(getFile())
+  // https://userbase.com/docs/sdk/get-databases/
```

Comment:
This is to get a list of all the user's databases. One database in entries for instance, another is settings. But you shall never use these directly. You should use selectors and actions to interact with them.